### PR TITLE
Fixes incorrect mount path for game

### PR DESCRIPTION
**Pull request by ultimaweapon**

### CODE REVIEW

File: src/kernel/src/main.rs [hunk 1]
```diff
@@ -215,18 +214,31 @@ fn start() -> Result<(), KernelError> {
         return Err(KernelError::MountFailed(path.into(), e));
     }
 
-    let path = VPathBuf::try_from(format!(
-        "/mnt/sandbox/{}_000/app0",
-        args.game.file_name().unwrap().to_str().unwrap()
-    ))
-    .unwrap();
+    // TODO: Check permission of these paths on the PS4.
+    let paths = [vpath!("/mnt/sandbox"), vpath!("/mnt/sandbox/pfsmnt")];
```
Comment (SuchAFuriousDeath) [inline src/kernel/src/main.rs:218]: I originally mkdir'd these paths too, but I though that they always exist inside tmpfs

**Review by SuchAFuriousDeath — APPROVED** [review]

**Comment by SuchAFuriousDeath** [comment]
And then we just change the root of the process to /mnt/sandbox/pfsmnt/{}-app0, right? I wasn't so far off :sweat_smile:

File: src/kernel/src/main.rs (hunk 1 above)
Comment (ultimaweapon) [reply src/kernel/src/main.rs:218]: All files in the tmpfs will live in the memory.

**Comment by ultimaweapon** [comment]
> And then we just change the root of the process to /mnt/sandbox/pfsmnt/{}-app0, right? I wasn't so far off 😅

It should be a nullfs that mount this path.

**Comment by ultimaweapon** [comment]
Seems like there is a bug in Actions because it should add `S-ready`.

**Comment by SuchAFuriousDeath** [comment]
Github battle part 2 incoming? :D

**Comment by SuchAFuriousDeath** [comment]
By the way, I would've implemented tempfs lookup already, but I'm not sure I know how. It seems like we need to get the actual type from dyn VnodeBackend

**Comment by SuchAFuriousDeath** [comment]
Wait what? That makes no sense. It's actually fine, I just got confused yesterday

**Pull request merged**